clients/cmd/logstash/Gemfile.lock

```diff
@@ -0,0 +1,165 @@
+PATH
+  remote: logstash/logstash-core-plugin-api
```

Comment:
Should the lock file be committed ? I'm not a ruby expert.

Reply:
it should not be commited